src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -142,6 +140,17 @@ else if (dateArray.length == 3) {
                     newReference.setMonument(monument);
 
                     monument.getReferences().add(newReference);
+                    break;
+                case 25: // Image filename
+                    value = formatJpgImageFileName(value);
```

Comment:
Why are we assuming everything is a jpg?

Reply:
All of the images I looked at from the initial set were jpgs. I wasn't sure what the valid assumption to make was, so I just chose jpg.

Reply:
While this works now it won't work in the future as I don't think we should restrict users to uploading jpgs. Imagine how annoying that would be

Reply:
Oh no, definitely not. I did it because some of the image filenames in the Initial Dataset spreadsheet didn't have file extensions on them, so the Object URL that would have gotten generated wouldn't have actually existed and the Monument would have no image.